### PR TITLE
Adding Unequal Variance to Signature Analysis T-Test

### PR DESCRIPTION
**Pull request by gwaygenomics**
ef98d58511a0750fc050d1af55f377e9dd4770ff is the major change of this pull request.

I tracked this error down after failing to align t-test results in  subpanels b, d, and f to the results shown in a, c, and e (respectively). 

A t-test with equal variance assumption will fail to identify cases found in b. 

Another minor change is plotting panels b and d with the test-set features only. Previously, panels a and b were calculated using the test set and panels b and d were generated using the training set. Now, b and d use the test set.

I reran the pipeline after these changes and all files here are updated.

## Updated Figure

full_separation_plot

### CODE REVIEW

File: 10.gene-expression-signatures/1.separate.ipynb [hunk 1]
```diff
@@ -54,7 +54,7 @@
     "    b_activation = feature_series[feature_series.index.isin(group_b_ids)]\n",
     "    \n",
     "    # Perform t-test on two groups\n",
-    "    t_stat, t_p = ttest_ind(a_activation, b_activation)\n",
+    "    t_stat, t_p = ttest_ind(a_activation, b_activation, equal_var=False)\n",
```
Comment (cgreene) [inline 10.gene-expression-signatures/1.separate.ipynb]: 👍

File: 10.gene-expression-signatures/0.download-validation-data.ipynb [hunk 2]
```diff
@@ -100,7 +100,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "url = \"https://www.nature.com/articles/sdata201733/tables/3\"\n",
+    "url = \"https://www.nature.com/articles/sdata201733/tables/4\"\n",
```
Comment (cgreene) [inline 10.gene-expression-signatures/0.download-validation-data.ipynb]: 🙀
